homeassistant/components/overkiz/strings.json

```diff
@@ -72,8 +72,8 @@
           "preset_mode": {
             "state": {
               "auto": "Auto",
-              "comfort-1": "Comfort 1",
-              "comfort-2": "Comfort 2",
+              "comfort-1": "Comfort -1",
+              "comfort-2": "Comfort -2",
```

Comment:
Can you elaborate on why this is renamed? Is this how Somfy/Atlantic call the presets in their apps?

Reply:
Yes, you basically define target temperature for Comfort mode on your heater, and the app allows to choose also Comfort -1°C & Comfort -2°C

![cozytouch](https://github.com/user-attachments/assets/70f8f491-7b93-48d2-9042-6d0905e53560)

Reply:
Are you sure it is -1°C and -2°C? To me it looks like they just incremented the numbers by 1 for the comfort settings. That is the reason we did call it Comfort, Comfort 1 and Comfort 2 .

Reply:
Yes, I am. 

Here is a [link to some explanation (in French)](https://assistance-pro.thermor-pro.fr/hc/fr/articles/4402713496210-Qu-elles-sont-les-fonctions-disponibles-sur-l-application-Thermor-Cozytouch-pour-piloter-à-distance-des-appareils-de-chauffage-équipés-d-une-interface-Cozytouch)

It also makes sense from picture I shared (temperature is decreasing as you drag down).

Reply:
Should we then add the *C to make it more clear?

Reply:
Done !